CoreScriptingView/src/au/gov/asd/tac/constellation/views/scripting/ScriptingViewPane.java

```diff
@@ -439,4 +416,49 @@ private void executeScript() {
             DialogDisplayer.getDefault().notify(notifyDescriptor);
         }
     }
+
+    @PluginInfo(pluginType = PluginType.CREATE, tags = {"CREATE"})
+    public static class LoadScriptPlugin extends SimplePlugin {
```

Comment:
I could be wrong, but this feels more like an import rather than a create?

Reply:
@aldebaran30701 I think `IMPORT` is a bit closer to the action as you can load a script from a file